tailwind.config.js

```diff
@@ -7,7 +7,13 @@ module.exports = {
         josefin: ["Josefin Slab", "serif"],
       },
     },
+    screen: {
+
+      'sm':'425px',
+      'md':'768px',
+      'xl':'1024px',
```

Comment:
why you used sm,md,xl,2xl  , cant we use lg too instead of having 2xl

Reply:
but this is work well